addon/index.js

```diff
@@ -1,7 +1,23 @@
 import Ember from 'ember';
 
-const { isNone } = Ember;
+const { isNone, warn } = Ember;
 
 export default function testSelector(key, value) {
   return isNone(value) ? `[data-test-${key}]` : `[data-test-${key}="${value}"]`;
 }
+
+export function ts(literals, ...substitutions) {
+  let interpolated = literals[0] + substitutions.map((_, i) => substitutions[i] + literals[i + 1]).join('');
+  if (interpolated.indexOf('%') === -1) {
+    warn(`[ember-test-selectors] "${interpolated}" contains no test selectors. Please use the % prefix for test selectors (e.g. ts\`%foo\`).`, false, {
+      id: 'ember-test-selectors.missing-test-selector-prefix',
+    });
```

Comment:
Why is this warning necessary? It's totally fine if it doesn't contain any selectors I'd say. This might be something that would be better implemented in ESLint where people can opt-in/out of it and decide on the warning/error level etc. themselves?